lib/commands.js

```diff
@@ -190,7 +190,7 @@ var operations = module.exports.operations = {
       return context.next();
     }
 
-    var questions = template.questions(context.newTemplate, { defaults: context.oldParameters });
+    var questions = template.questions(context.newTemplate, { defaults: context.oldParameters }, { region: context.region });
```

Comment:
If we always provide this 3rd arg, then all parameters will always get encrypted. Could we make this part of `context.overrides` instead? 

It's kind of a weird matrix: 

```js
commands.create('staging', 'template', { kms: true }, callback); // encrypt with default key id
commands.create('staging', 'template', callback); // no encryption
commands.create('staging', 'template', { kms: 'my-key-id' }, callback); // encrypt with provided key id
```